### PR TITLE
Feature/world UUID loading

### PR DESCRIPTION
**Pull request by Warriorrrr**
#### Description: 

____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [ ] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

**Comment by Warriorrrr** [comment]
Tested with a flatfile db and it correctly added the uuid and name to the world in the db, getWorld related errors also stopped. Still need to do some testing with an sql db though

File: src/com/palmergames/bukkit/towny/listeners/TownyWorldListener.java [hunk 1]
```diff
@@ -64,20 +66,38 @@ private void newWorld(World world) {
...
 				Bukkit.getServer().getPluginManager().getPlugin("DungeonsXL") != null) {
 			townyWorld.setUsingTowny(false);
 			townyWorld.save();
 		}
 	}
 
+	/**
+	 * Attaches UUIDs to worlds that might not have them
+	 */
+	private void checkWorlds() {
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/listeners/TownyWorldListener.java:91]: This method probably should have the DXL test we do to exclude dungeons XL worlds, no?

Comment (Warriorrrr) [reply src/com/palmergames/bukkit/towny/listeners/TownyWorldListener.java:91]: This just adds UUIDs to existing townyworlds that don't have one, when we create a townyworld for a DXL world it also has a uuid.

File: src/com/palmergames/bukkit/towny/db/TownySQLSource.java [hunk 2]
```diff
@@ -593,15 +594,35 @@ public boolean loadWorldList() {
 
 		TownyMessaging.sendDebugMsg("Loading World List");
 
+		// Check for any new worlds registered with bukkit.
+		if (plugin != null) {
+			for (World world : plugin.getServer().getWorlds())
+				universe.newWorld(world);
+		}
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/db/TownySQLSource.java:601]:
```suggestion
		for (World world : Bukkit.getServer().getWorlds())
			universe.newWorld(world);
```

**Review by LlmDl — APPROVED** [review]

**Pull request merged**